.github/actions/spell-check/expect.txt

```diff
@@ -1982,6 +1571,7 @@ THISCOMPONENT
 THotkey
 thumbcache
 TILEDWINDOW
+tilovell
```

Comment:
This is a username. Why are we removing from names.txt and adding in expect.txt?